core/src/main/scala/sttp/client3/RequestT.scala

```diff
@@ -233,6 +233,12 @@ case class RequestT[U[_], T, -R](
 
   def tag(k: String): Option[Any] = tags.get(k)
 
+  private val disableAutoDecompressionKey = "disableAutoDecompression"
```

Comment:
maybe let's add a comment that we're using tags to keep bin compat but this will be promoted to an option in the future?